modules/ROOT/pages/ip-whitelist.adoc

```diff
@@ -7,68 +7,66 @@ endif::[]
 [width="100%", cols="5,15"]
 |===
 >s| Policy Name | IP Whitelist
->s|Summary      | Allows request only from a list or range of IP addresses
+>s|Summary      | Allows access to request from a list or range of specified IP addresses
 >s|Category | Security
->s| Since Mule Version | 3.8.0
+>s|First Mule version available | 3.8.0
 .1+>.^s| Returned Status Codes
 | 403 - IP is rejected
 |===
 
 
-The IP Whitelist policy allows a list or range of IP addresses to access and interact with a configured API endpoint.
+The IP Whitelist policy allows a list or range of IP addresses to access a configured API endpoint.
 
-==  Whitelisting IP addresses to access your API endpoint
+The IP Whitelist policy allows access to a protected resource when a match is found between a source IP (specified when configuring the policy) and a list of individual IPs or range of IPs.
 
-You can set up access in the following ways:
+Based on your business need, you can specify a DataWeave 2.0 expression for Mule runtime engine 4 (Mule)) or MEL expression for Mule 3.
 
-* Access based on the IP address of the request
-* Access based on the origin IP address of the x-Forwarded-For header
 
-== Whitelisting Based on the IP address of the Request
+[[specifying_ip_address]]
+== Specifying IP Addresses to Whitelist in Your Policy 
 
-. Click the instance number of an API.
-+
-. On the API dashboard, click Policies.
-+
-The list of any applied policies appears.
-+
-. In Apply New Policy.
-. In Select Policy, choose IP Whitelist, and click Configure Policy.
-. Specify an address or a range or addresses as follows:
-+
-* Define a specific IP address. For example, 192.168.1.1.
-* Define a subset of addresses by identifying a subnet mask, such as 192.168.3.1/30. This includes the consolidated range: 192.168.3.0 - 192.168.3.3.
-* Define a whole range of IP addresses by stating the relevant octets of the IP address you want to permit. For example, setting `192.168` will include IP addresses from 192.168.0.0 to 192.168.255.255.
+You can specify the IP addresses to approve in the policy using the following ways:
 
-To whitelist a list of IP addresses:
+* Whitelist access based on the IP address of the request
+* Whitelist access based on the origin IP address that is determined by resolving a specific DataWeave or MEL expression, such as the x-Forwarded-For header or the HTTP header.
+* Any other origin
 
-. Enter one or more IP addresses in *Whitelist*.
-+
-image::ip-whitelist-051a0.png[height=353,width=377]
-+
-. Click +.
-+
-The greyed-out IP address and a deletion control appears below the Whitelist text entry field.
-+
-//image::ip-whitelist-1b671.png[height=353,width=377]
-image::ip-whitelist-1b671.png[height=475,width=500]
+=== Whitelisting Access Based on the IP address of the Request
+
+To whitelist access based on the IP address of the request, you can:
+
+* Define a specific IP address by enumerating it in the white space, for example, `192.168.1.1`.
```

Comment:
"white space" ? You mean the whitelist field, right?

Reply:
Ditto my previous edit about "Define".
ALSO...you used "white space" earlier, and I thought nothing of it...but if it should be "the whitelist field," as Nahuel suggests, please fix that both here and earlier.

Reply:
changed it to the whitelist field. Not sure where white space came from. :-) Legacy stuff. Done! Thanks!